.github/ISSUE_TEMPLATE/add_dataset.yml

```
@@ -0,0 +1,122 @@
+name: Add dataset
+description: Suggest a new Library, Archive or Museum dataset to include in the Hugging Face Hub.
+title: "Add dataset: [dataset_name_in_snake_case]"
+labels: ["candidate-dataset"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for suggesting a dataset!
+  - type: input
+    id: data-url
```

I would also propose to make the IDs as simple as possible: these can be passed as URL keyword arguments besides `template=add_dataset.yml` (so it's implicit they refer to adding a dataset)
```suggestion
    id: url
```